packages/icons/icons.json

```diff
@@ -4009,5 +4009,26 @@
         "tags": "model, ml",
         "group": "action",
         "codepoint": 62269
+    },
+    {
+        "displayName": "Lengthen",
+        "iconName": "lengthen",
```

Comment:
can we rename these to `"lengthen-text"` and `"shorten-text"`, respectively?

I feel like they are specific to text, and I'd like to leave scope for other icons which pertain to lengthening / shortening real-world measurements rather than text.

you'll have to rename the icon files to match these new names.